packages/flutter/lib/src/widgets/app.dart

```diff
@@ -1648,7 +1649,7 @@ class _WidgetsAppState extends State<WidgetsApp> with WidgetsBindingObserver {
                   child: Localizations(
                     locale: appLocale,
                     delegates: _localizationsDelegates.toList(),
-                    child: title,
+                    child: SelectionArea(child: title),
```

Comment:
I wonder whether we should make this to be per modalRoute thing

Reply:
Ahh, that would probably solve the problem of being able to select text on different routes...

Reply:
Where would you put this in the router/routing/navigator to essentially wrap each route in its own?

Reply:
I was imagine we can probably put this in _ModalScope, but I am also worry about whether there is a use case when user want to select across different routes?

Reply:
To construct one: One bigger desktop screens you may have a two column layout where the left and right side show different Navigator stacks. Do we need to support selecting across both?

Reply:
Ahh, this works great actually, solves the problem of highlighting old routes. I'm not sure what the use case would be for straddling different modal routes TBH